### PR TITLE
873: Update faq links to sumo

### LINKED ISSUE #873: Update FAQ links & feedback to point to support.mozilla.com articles
### Why/User Benefit/User Problem
Right now we're not able to adequately support our users well with Discourse and our obligatory email address in the app store. The Firefox Support team is stepping in to better cover our product support needs, which requires some updates on our side.

Mapping here: https://docs.google.com/spreadsheets/d/16bKy18uXTGHMAG7UVZbVRoxd_RNgKGaKtmbnR_D9LFQ/edit#gid=0

### PR DESCRIPTION
**Pull request by eliserichards**
Fixes #873

## Testing and Review Notes

I've crossed out the links that I've updated in the [spreadsheet](https://docs.google.com/spreadsheets/d/16bKy18uXTGHMAG7UVZbVRoxd_RNgKGaKtmbnR_D9LFQ/edit#gid=0).

### new FAQ landing page: 
image

### learn more and description removed from filter
image

### link to security removed from onboarding confirmation screen
image

## To Do

- [x] double check the original issue to confirm it is fully satisfied
- [x] add testing notes and screenshots in PR description to help guide reviewers
- [x] add unit tests
- [x] request the "UX" team perform a design review (if/when applicable)
- [x] make sure CI builds are passing (e.g.: fix lint and other errors)
- [ ] check on the [accessibility](https://mozilla-lockwise.github.io/lockwise-android/accessibility/) of any added UI

### CODE REVIEW

File: app/src/main/java/mozilla/lockbox/view/AutofillFilterFragment.kt [hunk 1]
```diff
@@ -38,7 +36,11 @@ import mozilla.lockbox.presenter.FilterView
 class AutofillFilterFragment : DialogFragment(), FilterView {
     val adapter = ItemListAdapter(ItemListAdapterType.AutofillFilter)
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```
Comment (jhugman) [inline app/src/main/java/mozilla/lockbox/view/AutofillFilterFragment.kt]: :+1:

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -228,12 +228,8 @@
     <string name="finish">Finish</string>
     <!-- This description of the app functionality is shown on the final onboarding screen -->
     <string name="access_description">Access logins saved to Firefox from your phone</string>
-    <!-- This is an accessible content description for the security information link text. %1$s will be replaced with application name -->
-    <string name="security_content_description">This is a link to a description of security practices in %1$s.</string>
     <!-- This description of the app security is shown on the final onboarding screen. %s will be replaced with the security link. -->
-    <string name="security_description">Sync between devices with secure %s</string>
-    <!-- This text is displayed as a link in the security description. -->
-    <string name="security_link">256-bit encryption</string>
+    <string name="security_description">Sync between devices with secure 256-bit encryption</string>
```
Comment (jhugman) [inline app/src/main/res/values/strings.xml]: Exhibit A for how a simple design change causes lots of code to be written or deleted.

Comment (eliserichards) [reply app/src/main/res/values/strings.xml]: 🙏

**Comment by eliserichards** [comment]
> Welcome Screen - Tapping [Learn More] crashes the app (I think this was happening to me on 2.0.0 as well though, fwiw)

@jhugman 's awesome error logging strikes again!: ```Caused by: java.lang.IllegalStateException: Cannot route from mozilla.lockbox:id/fragment_welcome to mozilla.lockbox:id/fragment_webview.```

**Comment by eliserichards** [comment]
> Settings Support - Not covered in this PR, but noticed that our link hover/press copy is no longer present on this type of link style (same as on the welcome screen link) nor does the tap target seem to be consistent (I tend to have to press it multiple times before the page will load) A consideration for a follow-up issue... 

Filed a followup: 

> Also just noticed that with build 4927, it's stating that the app version is 1.1.3. Is that expected?

Yep. Forgot to change the build # on master. Fixed now.

> Edit Login - Tapping [Learn More] brings us to the lockwise SUMO landing page, which doesn't necessarily match expectations @sandysage should we just consider removing this link (for context, it's around the "Entry names are not editable at this time" message) or should we create an article to address it in SUMO?

👌

**Comment by eliserichards** [comment]
Changed the uri so that it points to the top of the Getting Started page instead of `#w_step-1-sign-up-for-firefox-accounts`:

image

**Pull request merged**